plugin_runner/tests/test_plugin_installer.py

```diff
@@ -0,0 +1,117 @@
+import json
+import tarfile
+import tempfile
+import zipfile
+from pathlib import Path
+
+from pytest_mock import MockerFixture
+
+from plugin_runner.plugin_installer import install_plugins, uninstall_plugin
+
+
+def _create_tarball(name: str) -> Path:
+    # Create a temporary tarball file
+    temp_dir = tempfile.tempdir
+    tarball_path = Path(f"{temp_dir}/{name}.tar.gz")
```

Comment:
`Path` supports the `/` operator so that you don't have to embed the path separators in strings.

```suggestion
    tarball_path = Path(temp_dir) / f"{name}.tar.gz"
```

Reply:
Yeah that's better, making the change globally